### PR TITLE
add: `distrobox`

### PR DESCRIPTION
**Pull request by cat-master21**

### CODE REVIEW

**Comment by cat-master21** [comment]
I can verify these commands work for me:
Screenshot
And just to remember Pacstall will link the /usr/bin so distrobox will be $PATH.

File: packages/distrobox/distrobox.pacscript [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+description="Use any linux distribution inside your terminal."
+maintainer="cat-master21 <96554164+cat-master21@users.noreply.github.com>"
+url="https://github.com/89luca89/distrobox/archive/refs/tags/${version}.tar.gz"
+depends="podman"
+gives="distrobox"
+breaks="${gives} ${gives}-git ${gives}-bin"
+hash="86c7a6515d8548e258b4acb152e4a622f671c4ffcf1cc1a36d05b5f56bc6a0aa"
+prepare() {
+  sudo mkdir -p "/usr/src/pacstall/distrobox/usr"
+}
```
Comment (wizard-28) [inline packages/distrobox/distrobox.pacscript:13]:
```suggestion
}

build() {
  true
}
```

**Review by wizard-28 — CHANGES_REQUESTED** [review]

**Comment by wizard-28** [comment]
@cat-master21 dependency issue on Ubuntu, check the CI

**Comment by cat-master21** [comment]
Thinking about Podman dependency and just time to maintain this.

**Pull request closed without merging**